### PR TITLE
PE-76  | Rework Profile Onboarding

### PR DESCRIPTION
**Pull request by txt020**
**What I did:**

- Removed city and birthday fields.
- Users can now upload a profile picture when they first sign up.
- Users can edit their profile picture and will be able to view it on profile screen and side menu.

**How to test**

1. Pull from my branch
2. Create a new account or use an existing one.
3. Navigate to profile screen and hit edit profile.
4. Take a picture or select from gallery.
5. Make sure picture and your current edits show up on profile screen and side menu.
6. Make sure your picture was uploaded to mongo db and firebase. Match the ids accordingly.
7. Test again, a new account or existing one, which ever you did not test first.

### CODE REVIEW

**Review by dominikyz — APPROVED** [review]
Works well for me. Good Job!

File: lib/screens/profile_onboarding_screen.dart [hunk 1]
```diff
@@ -21,21 +27,18 @@ class ProfileOnboarding extends StatefulWidget {
 
 class _ProfileOnboardingState extends State<ProfileOnboarding> {
   final GlobalKey<FormState> _formKey = GlobalKey<FormState>();
+  String mediaPath = '';
+  ImagePicker picker = ImagePicker();
+  bool _isUploading = false;
```
Comment (tonynguyen98) [inline lib/screens/profile_onboarding_screen.dart]: If you're not using _isUploading, please remove it.

File: lib/screens/profile_onboarding_screen.dart [hunk 2]
```diff
@@ -121,6 +83,46 @@ class _ProfileOnboardingState extends State<ProfileOnboarding> {
...
   }
 
+  Future<List<String>> uploadFiles() async {
+    List<String> _mediaUrls = [];
+
+    setState(() {
+      _isUploading = true;
+    });
+
+    /// iterates through _mediaPath list and upload one by one.
```
Comment (tonynguyen98) [inline lib/screens/profile_onboarding_screen.dart]: dead comment here

**Review by tonynguyen98 — APPROVED** [review]
Looks good, and it seems to be working. Just some minor changes.

**Review by davidsquines — CHANGES_REQUESTED** [review]
Screen Shot 2021-04-28 at 10 42 19 AM
On this bottom bar when picking whether to take a new photo or pick photo from gallery, can you make the text fields bigger and the pop up container bigger? It is hard to see from the bottom of the screen.

File: lib/screens/profile_onboarding_screen.dart (hunk 1 above)
Comment (txt020) [reply lib/screens/profile_onboarding_screen.dart]: @tonynguyen98 

_isUploading is being used.

**Comment by txt020** [comment]
@davidsquines  @tonynguyen98 Requested changes made.

**Review by davidsquines — APPROVED** [review]
Works as intended

**Pull request merged**